build-system/extensions-versions-config.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the license.
+ */
+
+var extensionsVersions = {
+  'amp-sticky-ad': ['1.0'],
+};
```

Comment:
can't we just build this dynamically by reading extensions/${extension-name}? do you think explicitly listing it down is worthwhile?